### PR TITLE
ResNet3D VAE fix

### PR DESCRIPTION
**Pull request by pyushkevich**
Hi there,

I was just playing around with the package and found a typo in ResNet3D VAE constructor that prevents network for working with inputs where x,y,z dimensions are different (e.g., 56, 64, 48). 

Thanks!
Paul

### CODE REVIEW

**Comment by SergiosKar** [comment]
LGTM. Thanks for finding and fixing the bug. Feel free to merge.

File: lib/medzoo/ResNet3D_VAE.py [hunk 1]
```diff
@@ -235,7 +235,7 @@ def forward(self, x):
         x = x.view(-1, self.linear_in_dim)
         x = self.linear_1(x)
         mu = x[:, :self.split_dim]
-        logvar = torch.log(x[:, self.split_dim:])
+        logvar = x[:, self.split_dim:]
```
Comment (SergiosKar) [inline lib/medzoo/ResNet3D_VAE.py:238]: That's correct. The `log` here is misplaced because it is indirectly assumed in the latent vector `x`. I summon @black0017 for a second opinion

Comment (black0017) [reply lib/medzoo/ResNet3D_VAE.py:238]: I see! At the moment of writing this I was not aware of the fact that `log` is redundant. Thanks!!!

**Review by SergiosKar — APPROVED** [review]

File: lib/medzoo/ResNet3D_VAE.py [hunk 2]
```diff
@@ -254,7 +254,7 @@ class ResNet3dVAE(BaseModel):
     def __init__(self, in_channels=2, classes=4, max_conv_channels=256, dim=(64, 64, 64)):
         super(ResNet3dVAE, self).__init__()
         self.dim = dim
-        vae_in_dim = (int(dim[0] >> 3), int(dim[1] >> 3), int(dim[0] >> 3))
+        vae_in_dim = (int(dim[0] >> 3), int(dim[1] >> 3), int(dim[2] >> 3))
```
Comment (black0017) [inline lib/medzoo/ResNet3D_VAE.py:257]: this is definitely a typo. now it should work with any volume dim!!! thanks!!

**Review by black0017 — APPROVED** [review]
'log' is unnecessary for vae. Now it supports any volume size also!!!Thanks
